### PR TITLE
Change some translated strings

### PR DESCRIPTION
**Pull request by antoin-m**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | @xBorderie wasn't really happy with what I did there https://github.com/PrestaShop/PrestaShop/pull/7393 so here we go again :smile: 
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |

### CODE REVIEW

File: controllers/admin/AdminImportController.php [hunk 1]
```diff
@@ -3002,15 +3002,25 @@ protected function customerImportOne($info, $default_language_id, $id_lang, $sho
...
-                array(
-                    $info['email'],
-                    (isset($info['id']) && !empty($info['id']))? $info['id'] : 'null',
-                    ($validateOnly?'validated':'saved')
-                ),
-                'Admin.Advparameters.Notification'
-            );
+            if ($validateOnly) {
+                $this->errors[] = $this->trans(
+                    '%1$s (ID: %2$s) cannot be validated.',
```
Comment (xBorderie) [inline controllers/admin/AdminImportController.php]: Easier to translate: `Email address %1$s (ID: %2$s) cannot be validated`.

File: controllers/admin/AdminImportController.php [hunk 2]
```diff
@@ -3002,15 +3002,25 @@ protected function customerImportOne($info, $default_language_id, $id_lang, $sho
...
+                    array(
+                        $info['email'],
+                        (isset($info['id']) && !empty($info['id']))? $info['id'] : 'null',
+                    ),
+                    'Admin.Advparameters.Notification'
+                );
+            } else {
+                $this->errors[] = $this->trans(
+                    '%1$s (ID: %2$s) cannot be saved.',
```
Comment (xBorderie) [inline controllers/admin/AdminImportController.php]: Easier to translate: `Email address %1$s (ID: %2$s) cannot be saved`.

**Review by xBorderie — APPROVED** [review]
Thank you!

**Comment by aleeks** [comment]
Thank you @antoin-m

**Comment by AlexEven** [comment]
For future reference, if we have to introduce new variables to an existing string, better use the new %naming_convention%, rather than %s! It will provide more context to the translators :)
Thanks!

**Pull request merged**